File: server/router/feedback/feedback.go
package feedback

import (
	"github.com/flipped-aurora/gin-vue-admin/server/middleware"
	"github.com/gin-gonic/gin"
)

type FeedbackRouter struct{}

func (r *FeedbackRouter) InitFeedbackRouter(Router *gin.RouterGroup) {
	g := Router.Group("feedback").Use(middleware.OperationRecord())
	gWithout := Router.Group("feedback")
	{
		g.DELETE("deleteFeedback", feedbackApi.DeleteFeedback)
		g.DELETE("deleteFeedbackByIds", feedbackApi.DeleteFeedbackByIds)
	}
	{
		gWithout.GET("findFeedback", feedbackApi.FindFeedback)
		gWithout.GET("getFeedbackList", feedbackApi.GetFeedbackList)
	}
}

File: web/src/api/feedback.js
import service from '@/utils/request'

export const deleteFeedback = (data) => {
  return service({
    url: '/feedback/deleteFeedback',
    method: 'delete',
    data
  })
}

export const deleteFeedbackByIds = (data) => {
  return service({
    url: '/feedback/deleteFeedbackByIds',
    method: 'delete',
    data
  })
}

export const getFeedbackList = (params) => {
  return service({
    url: '/feedback/getFeedbackList',
    method: 'get',
    params
  })
}

export const findFeedback = (params) => {
  return service({
    url: '/feedback/findFeedback',
    method: 'get',
    params
  })
}
